### ISSUE TITLE
License hyperlink on anaconda.org not working

### ISSUE THREAD

**Issue by leofang** [conda-forge/cuquantum-python-feedstock#4]
This hyperlink on anaconda.org does not correctly jump to `license_url` set in `meta.yaml`:
<img width="737" alt="截圖 2022-01-05 下午3 23 11" src="https://user-images.githubusercontent.com/5534781/148284006-15f9b73b-a2d6-480d-bc47-6f0fa54c693c.png">

**Comment by jakirkham** [comment]
Interesting this says `BSD-3-Clause` and links to the OSI page. However the `license_url` seems to be linking to a license that doesn't look like `BSD-3-Clause`. Should we update that as well?

**Comment by leofang** [comment]
I think this is the same bug seen elsewhere, except for this case the license is an open one, not a proprietary one, so it fired up a text search on OSI. (In other places, a search of proprietary licenses would fail.)

**Comment by jakirkham** [comment]
It could be. In either case it seems like it shouldn't say `BSD-3-Clause` right? Once that is updated it should be easy to confirm whether the issue is still there.

**Comment by leofang** [comment]
Yes I think it should say that. For example, see CUB: https://anaconda.org/conda-forge/cub which is released under BSD-3. It's a valid OSI-approved name. (And cuQuantum Python is released also under BSD-3, see https://docs.nvidia.com/cuda/cuquantum/license.html#nvidia-cuquantum-python.)

**Comment by leofang** [comment]
cc: @chenghlee
